include/z64skybox.h

```diff
@@ -66,6 +66,10 @@ typedef struct SkyboxFile {
 
 extern SkyboxFile gNormalSkyFiles[];
 
+// Tests if the skybox palette for the given skybox is assigned to color index 0-127.
+// If false, it is assigned to color index 128-255 instead.
+#define IS_NORMAL_SKYBOX_IMAGE_COLOR_INDEX_0(skyboxIndex) ((skyboxIndex & 1) ^ ((skyboxIndex & 4) >> 2))
```

Comment:
```suggestion
#define IS_NORMAL_SKYBOX_IMAGE_TLUT_SPLIT_LO(skyboxIndex) ((skyboxIndex & 1) ^ ((skyboxIndex & 4) >> 2))
```
using the assets system's terminology